settings/base.py

```diff
@@ -355,3 +356,10 @@ def SOCIAL_AUTH_DEFAULT_USERNAME(u):
 # if True admin site will not be affected by the maintenance-mode page
 MAINTENANCE_MODE_IGNORE_ADMIN_SITE = True
 FORM_RENDERER = "django.forms.renderers.DjangoDivFormRenderer"
+
+# django-q2 settings
+# https://django-q2.readthedocs.io/en/stable/configure.html
+Q_CLUSTER = {
+    "name": "djangopackages",
+    "redis": env.str("REDIS_URL"),
+}
```

Comment:
We need to set some configurations here depending on the production server (ideally using environment variables) such as `workers`

We also need to decide if we want to disable task retries (e.g: `ack_failures`). If we want to enable it then we should also set `timeout` and `retry` values.

C.C: @jefftriplett

Reply:
I think the default is 4 workers (or at least the example config is 4 workers) which seems right to me. 

I don't like how they say they sleep/block though. We might just have to start with defaults and adjust. Worse case we build setup two queues so that we always have one available that won't ever block. At the moment just processing tasks out of band of a web node is an improvement so I don't think we have to optimize for make it more complex that we need.